### PR TITLE
Remove absolute dates from options

### LINKED ISSUE #7617: New dates UX is confusing
## Bug description

While the new ways to display dates does provide value, I think it introduced some confusion worth addressing. In particular, the way we display dates when collapsed.

## Expected behavior

IMHO should still display the relative time information (e.g. "Last 7 days") because:
a) It's generally easier to parse than a date range (at least for quick analysis, you may want to understand the specific range later).
b) It's confusing to select "Last 7 days" and then see a date range which is a different text (don't make me think).
c) The default date range is now a random date range which immediately makes me think something is outdated.

## How to reproduce

Create an insight.

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

## Additional context
CC @clarkus @corywatilo for input.

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Closes #7617

As per https://github.com/PostHog/posthog/issues/7617#issuecomment-991216175, we opted to remove absolute date ranges from the date dropdown.

Before

Screen Shot 2021-12-16 at 12 10 56 PM

After

Screen Shot 2021-12-16 at 12 11 00 PM

## How did you test this code?

Minor copy change. See that options don't have absolute dates anymore.

### CODE REVIEW

**Review by Twixes — DISMISSED** [review]
Honestly I find the new date picker pretty confusing, but not because of the menu – rather because of the main display always showing the absolute range.
Each time I open an insight I wonder "is it set to 'Last month' and that's just what the last month is – not like I can't just take a look at the calendar and subtract 30 days – or is it actually rigidly set to these dates". I really think when the filter is something like `-7d`, it should just say "Last week", and not try to outsmart me.
Zrzut ekranu 2021-12-17 o 12 01 28

**Comment by clarkus** [comment]
I agree with @Twixes - we can rely on the chart axes to represent the absolute date range. The controls benefit most from showing those relative ranges.

**Review by paolodamico — DISMISSED** [review]
I would actually consider doing this the other way around, keeping the relative + absolute  range in the list but the display should definitely be the relative time range. Feel free to just keep relative time ranges all across too.

**Comment by clarkus** [comment]
The absolute values really just expand the area needed by the menu way too much in my opinion. It's really overwhelming and hard to parse. We're always picking a value relative to the current date, so I feel like we can rely on the chart to represent absolutes and use relative values for providing input. Showing a value other than the one selected in the select input could also be technically complex and result in a weird experience for the user. Can we compromise and ship the following to see how it performs?

- Date control - consists of relative options. When selected, we show the same value in the control summary as was selected from the list. No absolute values in the control.
- Chart - shows the full range of time values in the x-axis - no relative values are displayed in here.

**Comment by alexkim205** [comment]
Thanks for the direction here @clarkus. For now I've reverted the look of the date filter dropdown to show relative options and controls, while keeping chart x axis values absolute. I've also left the `isDateFormatted` prop available so that we can revisit this option if we change our minds later.

**Review by paolodamico — APPROVED** [review]
